aws/resource_aws_redshift_cluster_test.go

```diff
@@ -59,42 +59,6 @@ func testSweepRedshiftClusters(region string) error {
 	return nil
 }
 
-func TestValidateRedshiftClusterDbName(t *testing.T) {
```

Comment:
should we replace this validation test (and the ones below) with something similar to your `tunnel_options` test in https://github.com/terraform-providers/terraform-provider-aws/pull/13022/files#diff-3c63318f039b0acd51d0033c0df31ee3R141 ? since I believe the other acceptance tests always pass valid values for these params

Reply:
ignore